src/pages/saksbehandling/steg/virkningstidspunkt/Virkningstidspunkt.tsx

```diff
@@ -85,7 +85,9 @@ const Virkningstidspunkt = (props: VilkårsvurderingBaseProps) => {
 
     form.watch((values) => {
         if (values.fraOgMed !== null && values.tilOgMed === null) {
-            form.setValue('tilOgMed', DateFns.endOfMonth(DateFns.addMonths(values.fraOgMed, 11)));
+            form.setValue('tilOgMed', DateFns.endOfMonth(DateFns.addMonths(values.fraOgMed, 11)), {
+                shouldValidate: true,
+            });
```

Comment:
litt 'rar' fiks for at man fjerner feil melding på TOM-feltet når man submitter først, så fyller ut FOM. etter litt testing, så er det ingen konsekvenser når man fyller dem ut på vanlig måte, eller etter submitting.